internals/secrethub/audit.go

```diff
@@ -63,13 +95,28 @@ func (cmd *AuditCommand) run() error {
 		return err
 	}
 
-	tabWriter := tabwriter.NewWriter(cmd.io.Stdout(), 0, 4, 4, ' ', 0)
-	header := strings.Join(auditTable.header(), "\t") + "\n"
-	fmt.Fprint(tabWriter, header)
+	paginatedWriter, err := cmd.newPaginatedWriter(os.Stdout)
```

Comment:
I am aware that it should be `cmd.io.Stdout()` but I'm not entirely sure why that option does not work. I'll look into it

Reply:
Actually, I'm not entirely sure why `cmd.io.Stdout()` is different from `os.Stdout`.

Reply:
> Actually, I'm not entirely sure why cmd.io.Stdout() is different from os.Stdout either.

What do you means?

Reply:
I mean that I'm not sure why they behave differently in this context. The former works as expected, the latter basically dumps all the entries on the screen and doesn't stop or allow scrolling.

Reply:
Been playing a little with it and I think I found the reason for this.

My suspicion is the following: some functionality inside the `cmd.NewPaginatedWriter` needs an `*os.File` for its interactive functionality. It obtains this by casting the provided `io.Writer` to  file:
```
writer.(*os.File)
```

This works if `os.Stdout` is used because it is an `*os.File`. However, `cmd.io.Stdout()` is not an `*os.File` it's an `ui.file`, so casting does not work... You can simulate this by (it panics):
```
io.Writer(cmd.io.Stdout()).(*os.File)
```
This casting problem is probably handled gracefully by reverting to non-interactive mode.

The underlying problem is that the `ui.IO` interface is not pretty. It is a weird abstraction that does some magic by embedding `*os.File` and creating some custom interfaces. That probably requires some work to make this work.

Reply:
After rewriting the `ui` package a bit based on your observation passing `cmd.io.Stdout()` works as expected.